site/src/docs/getting-started/tutorials/migrate-to-4.0.0.mdx

```diff
@@ -195,75 +195,95 @@ There are updates to lineheights to improve readability especially on Headings.
 
-The line heights of headings have been updated in the HDS Design Kit to improve readability, especially with larger titles. We have improved color black-60 for better contrast by changing it from #666666 to #595959.
+This HDS Design Kit update follows the changes deployed in the code for the 4.0.0 release of HDS. The major differences include significant changes to the **CookieConsent**, **Dropdown** and **Tag** components. The new **Select** and **Multiselect** components replace the old Dropdown.
 
-The HDS Design Kit in Figma has undergone a major restructuring: now, each component is placed on its own page. This new organisation makes it much easier to find components without having to guess which group they belong to. Each component will also have better documentation, which may include main component, subcomponents, specifications, variant examples, and usage & prototype examples, depending on the need and documentation status. We hope these changes will better explain the features of each component and how they can be used.
+The line heights of **headings** have been updated in the HDS Design Kit to improve readability, especially with larger titles. We have improved color **black-60** for better contrast by changing it from #666666 to #595959.
 
-### Components that need replacing
-[Dropdown] The Dropdown component is now deprecated and will be removed in future versions. Replace it with Select or Multiselect.
+The HDS Design Kit in Figma has undergone **a major restructuring**: now, each component is placed on its own page. This new organisation makes it much easier to find components without having to guess which group they belong to. Each component will also have better documentation, which may include main component, subcomponents, specifications, variant examples, and usage & prototype examples, depending on the need and documentation status. We hope these changes will better explain the features of each component and how they can be used.
 
-[Cookie consent] The CookieConsent component has been replaced with a new version, and it needs to be manually updated.
+#### Components that need replacing
 
-Deprecated components needs replacing, follow Figma guide about swapping components.
+- **[Dropdown]** The Dropdown component is now deprecated and will be removed in future versions. Replace it with Select or Multiselect.
+    
+- **[Cookie consent]** The CookieConsent component has been replaced with a new version, and it needs to be manually updated.
+    
 
-### Components that need manual checking
-[Headings] Line heights have been updated to improve readability. Please check that the layout does not break.
+Deprecated components needs replacing, follow [Figma guide](https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances) about swapping components.
```

Comment:
All external links should use external link format/component so that they'll have the icon + be accessible. Links leading to Github seem to not have been treated as external in the past, but at least the Figma site ones yes.

e.g. (this one doesn't have openInNewTab="true" , I'm not sure what's the guideline if links on site should or should not)  

```suggestion
Deprecated components needs replacing, follow <ExternalLink href="https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances">Figma guide</ExternalLink>  about swapping components.
```

Also importing it to the mdx (I would just copy everything from Migration guide 3.0.0)

import ExternalLink from '../../../components/ExternalLink';

Reply:
Also giving a more descriptive name to the link rather than "Figma guide"